infra/gcp/clusters/.gitignore

```
@@ -1,4 +1,16 @@
 .terraform/
-*.tfstate*
+.terraform.lock.hcl
+
+# .tfstate files
+*.tfstate
+*.tfstate.*
```

This matches both, unless there's a need to allow `*.tfstatefoo`?
```suggestion
*.tfstate*
```

---

Fixed